### DISCUSSION TITLE
[Script request] Romm.app

### DISCUSSION THREAD

**Discussion by mnlxr** [community-scripts/ProxmoxVE#429] [Request script]
### Application Name

Romm

### Website

https://romm.app/

### Description

self-hosted rom manager

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [x] This is not a game-related request.

**Comment by MickLesk** [comment]
You check the "not a game-related request." and requested a game rom manager? :smile:

**Reply by mnlxr** [reply]
Yes. Sorry.

**Reply by mnlxr** [reply]
I am back. It's not game tester. It's a game manager. For roms. So i click again that is not game-related request. Is that ok? 
Thank you

**Reply by mnlxr** [reply]
Some related info that might help :  [rommapp](https://github.com/rommapp/romm)

**Reply by MickLesk** [reply]
Not yet or in the next time. Nobody of us can maintain Game Scripts.

**Comment by wisie** [comment]
This [article](https://noted.lol/romm/) has me interested. Another vote :)
